src/main/java/org/pac4j/demo/dw/Pac4JDemoApplication.java

```diff
@@ -54,5 +55,7 @@ protected Result check() throws Exception {
                 return Result.healthy();
             }
         });
+
+        new DbServer();
```

Comment:
Maybe it would be better to put that in the `run` method and ensure it is properly closed on application end (via `env.lifecycle() `methods).